packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/modeledDataStreamQueryEditor/types.ts

```diff
@@ -0,0 +1,34 @@
+import { AssetPropertySummary, AssetModelPropertySummary } from '@aws-sdk/client-iotsitewise';
+
+type listPropertyOutputExtender = {
```

Comment:
Types should be written in pascal case, such as `ListPropertyOutputExtender`.

Reply:
Also, `Extender` sounds like it performs an action, yet this construct has no methods on it. What about something like `ExtendedAssetPropertySummaryProperties`?